### PR TITLE
fix: 修复打包core层源码中样式文件错误问题

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-react/src/components/tooltip/index.tsx [hunk 1]
```diff
@@ -18,8 +18,7 @@ import { TooltipSimpleTips } from './components/simple-tips';
 import { TooltipSummary } from './components/summary';
 import { TooltipRenderProps } from './interface';
 
-import '@antv/s2/dist/style.min.css';
-import '@antv/s2-shared/src/styles/tooltip/index.less';
+import './index.less';
```
Comment (wjgogogo) [inline packages/s2-react/src/components/tooltip/index.tsx:22]: 因为 `@antv/s2`是react的peer deps, 所以做打包时，这种导入方式不会将样式打包，最终生成的打包文件：
image
显然找不到这个文件， 所以会报错

而通过 less 文件中再 import 就会被打包进入react的样式文件，且没有上面的import问题

Comment (wjgogogo) [reply packages/s2-react/src/components/tooltip/index.tsx:22]: vue的行为与上述的”通过 less 文件中再 import“一致，所以没有问题

**Review by lijinke666 — APPROVED** [review]

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**